File: 02-Python/03-life-in-weeks.py
#!/usr/bin/python

your_age = int(input("What is your age? \n"))

def time_left(age):

    time_left = 90 - age

    months = time_left * 12
    weeks = time_left * 52
    days = time_left * 365

    print("\nYou have " + str(days) + " days, or " + str(weeks) + " weeks, or " + str(months) + " months left until age 90.")


time_left(your_age)

File: 01-JavaScript/03-life-in-weeks.js
const prompt = require("prompt-sync")({ sigint: true });

const yourAge = prompt("What is your age? ");

function timeLeft(age) {

    const currentAge = parseInt(age);

    const timeLeft = 90 - currentAge;

    const months = timeLeft * 12;
    const weeks = timeLeft * 52;
    const days = timeLeft * 365;

    console.log("You have " + days + " days, or " + weeks + " weeks, or " + months + " months left until age 90.");
}

timeLeft(yourAge);